.changeset/add_basic_cargo_workspace_support.md

```diff
@@ -0,0 +1,19 @@
+---
+default: minor
+---
+
+# Add basic Cargo workspace support
+
+If you have a `Cargo.toml` file in the working directory which represents a Cargo workspace containing fixed members, like:
+
+```toml
+[workspace]
+members = [
+  "my-package",
+  "my-other-package",
+]
+```
+
+then Knope will now treat each member like a package, assuming that there is a `Cargo.toml` file for each member.
```

Comment:
a note on this: 
I think it is always better to be explicit, rather than implicit when it comes to user interfaces, unless the interface itself tells you when an assumption it made turned out to be wrong.

Reply:
Yes, good point. According to the Cargo workspace docs, it's an error to point at a member which does not have a Cargo.toml file, so I just made it an error as well.